pkg/lang/ir/python.go

```diff
@@ -166,6 +167,24 @@ func (g Graph) compilePyPIPackages(root llb.State) llb.State {
 			llb.Local(flag.FlagBuildContext), llb.Readonly)
 		root = run.Root()
 	}
+
+	if len(g.PythonWheels) > 0 {
+		for _, wheel := range g.PythonWheels {
+			root = root.File(llb.Copy(
```

Comment:
We could mount the build context dir like here.

https://github.com/tensorchord/envd/pull/838/files#diff-e9007374abf264af40157329f8dd5adff0461ef389c98a8daa32eaf54b5db8adL165